src/health_misinfo_shared/fine_tuning.py

```diff
@@ -264,7 +264,7 @@ def get_model_by_display_name(display_name: str) -> TextGenerationModel:
 
 def get_video_responses(
     model, chunks: list[str], multilabel: bool = False, in_context_examples: str = ""
-) -> None:
+) -> Generator:
```

Comment:
I believe this could be a well-typed as Iterable (`Iterable[dict]` at least), bit clearer than a Generator IMO

Reply:
Yeah good point, `Generator` was a bit lazy on my part!

Reply:
I think this could be: `Generator[dict[str, Any], None, None]`

Does that sound okay?

Reply:
I don't see the aversion to Iterable (Generator subclasses Iterable iirc) but that too sounds okay

Reply:
> Generator subclasses Iterable iirc

^^ This is the reason I’ve preferred Generator here. I.e. the typing should be as specific as possible. We’re not actually using any of the extra features of a generator, so iterable is fine here.

Reply:
Okay – I’ve updated all instances of `Generator` with `Iterable`